dist/angular-sticky-table-header.js

```diff
@@ -109,7 +109,8 @@ angular.module('turn/stickyTableHeader', ['watchDom']).value('stickyTableHeaderO
             } else if (scope.stuck && scrollY < scope.offset.top) {
               scope.setStuck(false);
             }
-            scope.clone.css('left', scope.offset.left - scrollX);
```

Comment:
@bcherny . The horizontal scroll is taken care by setting position to absolute ALONG with left to 0. Both position: absolute & left: 0 has to go hand in hand in order to have the horizontal scroll work seamlessly.